src/main/jflex/VentoLexer.flex

```diff
@@ -36,8 +38,10 @@ import static com.intellij.psi.TokenType.WHITE_SPACE;
 
 
 WHITESPACE = [ \t\r\n]+
-COMMENT_START = \{\{#
-TRIMMED_COMMENT_START = \{\{#-
+OPEN_COMMENT_PHRASE = \{\{#
+CLOSE_COMMENT_PHRASE = #}}
+OPEN_COMMENT_TRIMMED_PHRASE = \{\{#-
+CLOSE_COMMENT_TRIMMED_PHRASE = -#}}
 JAVASCRIPT_START = \{\{>
 VARIABLE_START = \{\{
```

Comment:
JavaScript blocks and echo/interpolation can also trim with `{{-> … -}}` and `{{- … -}}` respectively